src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -120,6 +120,11 @@ def setup_multisite_replication(self, daemon_name=None, realm_name=None, zonegro
                                     zonegroup_endpoints=None, zone_name=None, zone_endpoints=None,
                                     username=None, cluster_fsid=None):
         multisite_instance = RgwMultisite()
+
+        # Handle empty cluster_fsid
+        if cluster_fsid == '':
+            cluster_fsid = None
+
```

Comment:
why is this?

Reply:
removed, was done to adjust some changes made in the multi-site service. Can be handled from there only, so removed